app/controllers/openid_token_proxy/callback_controller.rb

```diff
@@ -0,0 +1,20 @@
+module OpenIDTokenProxy
+  class CallbackController < ApplicationController
+    def handle
+      unless code = params[:code]
+        render nothing: true, status: :bad_request
```

Comment:
Ohh, alright. All the more reason to give it an error that they can report, but I still don't think it needs to be localized.